src/tests/FunctionalTests/iOS/common.targets

```diff
@@ -0,0 +1,29 @@
+<Project>
+
+  <!-- Redirect 'dotnet publish' to in-tree runtime pack -->
+  <Target Name="TrickRuntimePackLocation" AfterTargets="ProcessFrameworkReferences">
```

Comment:
We could have this in a common to all platforms targets file and then just set a property in each project with whatever the `PackageDirectory` value is.  Feels nit picky though.